.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 .cargo/config.toml
-/wasm
+wasm/
+dist/
```

Comment:
This will ignore any folder named `wasm/` or `dist/`. For example `src/foo/bar/dist/xyz.rs`. I don't think that's desirable.

Reply:
Fixed in the meantime